chainio/clients/elcontracts/writer.go

```diff
@@ -162,7 +162,10 @@ func (w *ChainWriter) RegisterAsOperator(ctx context.Context, operator types.Ope
 
 	w.logger.Infof("registering operator %s to EigenLayer", operator.Address)
 	opDetails := delegationmanager.IDelegationManagerOperatorDetails{
-		DeprecatedEarningsReceiver: gethcommon.HexToAddress(operator.EarningsReceiverAddress),
+		// Earning receiver has been deprecated but just to make sure we have something in contract
+		// We just force it to be operator address
+		// Any reward related setup is via RewardsCoordinator contract
```

Comment:
```suggestion
		// Earning receiver has been deprecated so we just use the operator address as a dummy value
		// Any reward related setup is now done via the RewardsCoordinator contract instead
```

Reply:
The comment was a bit confusing to me. I changed it, let me know if it doesn't make sense. If it's clearer, then please also change the other one.

Reply:
3a5f418fe1bb0ec640e531d17d1d88dbc19aa966